Casks/w/wacom-tablet.rb

```diff
@@ -9,7 +9,7 @@
 
   livecheck do
     url :homepage
-    regex(%r{/WacomTablet[._-]?v?(\d+(?:\.\d+)+(?:[_-]\d+)?)\.dmg}i)
+    regex(%r{/WacomTablet[._-]?v?(\d+(?:\.\d+)+(?:[_-]\d+)+(?:[a-zA-Z])?)\.dmg}i)
```

Comment:
```suggestion
    regex(%r{/WacomTablet[._-]?v?(\d+(?:\.\d+)+(?:[_-]\d+[a-z]?)?)\.dmg}i)
```

The current `regex` does not preserve the original behavior because the suffix is no longer optional. We can also simplify the character class to `[a-z]` since the `regex` is case-insensitive.